src/services/appDataService.ts

```diff
@@ -106,8 +84,9 @@ const appDataService = {
     return recordService.deleteRecord(ownerId, resourceId);
   },
 
-  fetchAllAppData(ownerId: string): Promise<FetchResponse> {
+  fetchAllAppData(ownerId: string, params: Params = {}): Promise<FetchResponse<AppData>> {
     const parameters = prepareSearchParameters({
+      ...params,
```

Comment:
Oh nice. What's the reason we're adding this here now?

Reply:
To pass such options like limit and offset. I am pretty sure we don't need them currently, but I thought why not keep the functions as similar as possible